gradle/libs.versions.toml

```diff
@@ -1,16 +1,20 @@
 [versions]
-managed-grpc = '1.47.0'
-managed-protobuf = '3.21.4'
+managed-grpc = '1.50.0'
+managed-protobuf = '3.21.8'
+managed-protobuf-gradle = '0.9.1'
 
-jackson-datatype-protobuf = '0.9.12'
+jackson-datatype-protobuf = '0.9.13'
 jaeger-core = '1.8.1'
 opentracing-grpc = '0.2.3'
 opentracing-mock = '0.33.0'
+micronaut-docs = '2.0.0'
 
 [libraries]
+
 managed-grpc-protobuf = { module = 'io.grpc:grpc-protobuf', version.ref = 'managed-grpc' }
 managed-grpc-stub = { module = 'io.grpc:grpc-stub', version.ref = 'managed-grpc' }
 managed-grpc-services = { module = 'io.grpc:grpc-services', version.ref = 'managed-grpc' }
+
 managed-protobuf-java = { module = 'com.google.protobuf:protobuf-java', version.ref = 'managed-protobuf' }
 managed-protobuf-java-util = { module = 'com.google.protobuf:protobuf-java-util', version.ref = 'managed-protobuf' }
 
```

Comment:
remove all the `micronaut-*` items here and use the `mn` catalog in dependency closures instead